example_site/src/components/AppProvider.js

```diff
@@ -62,14 +63,18 @@ export const AppProvider = ({ children }) => {
 
   const closeEditor = useCallback(() => {
     setIsEditorOpen(false);
-    history.push("/collections");
-  }, [setIsEditorOpen, history]);
+    if (purchasedVideo) {
```

Comment:
Ah, we're using the existence of a purchased video to figure out where to navigate to next. I think we can do something a little slicker! Since we are (probably) not using the `purchasedVideo` data anywhere else, what do you think about storing something like `exitEditorNextURL` instead of the video? Let's talk about how that might work!